tests/base/feature_importance_utils_test.py

```diff
@@ -42,8 +42,27 @@ def test_calculate_importance(iris_labeled_dataset):
 
 def test_bad_dataset_model(iris_random_forest, diabetes):
     ds, _ = diabetes
+    error_message = (
+        r'(In order to evaluate model correctness we need not empty dataset with the '
+        r'same set of features that was used to fit the model! But function received '
+        r'dataset with a different set of features!)'
+        # FIXME: 
+        # There is a problem, when I run this test locally, the 'validate_model' 
+        # function from the utils package as predicted raises DeepchecksValueError 
+        # with the message displayed above, but when it is run in the cloud (CI/CD) 
+        # the same function fails with different exception message (message below). 
+        # The reason for this is that when it is run on the cloud somewhy model instance 
+        # passed to the `calculate_feature_importance` does not have 'feature_names_in_' attribute
```

Comment:
Sounds like an issue with scikitlearn version. Can check if have different scikitlearn versions locally and in cloud and if random forest has changed this attribute

Reply:
Indeed, locally I use version 1.0.1, at the cloud (CI/CD) version 0.24 is used and that attribute was added at version 1.0

I removed my virtual env and reinstalled it with help of the `make env` and still got version 1.0.1/
I think that the python version which you use to install it could affect which package version will be used/

so, I suggest leaving this test as it is (but I updated the comment)

Reply:
I'm ok with not testing for the specific message. Is that what the test does? Because there are two strings there